src/test/java/net/snowflake/client/jdbc/FileUploaderSessionlessTest.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2012-2020 Snowflake Computing Inc. All right reserved.
+ */
+package net.snowflake.client.jdbc;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.util.*;
+import net.snowflake.client.jdbc.cloud.storage.StageInfo;
+import net.snowflake.common.core.RemoteStoreFileEncryptionMaterial;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/** Tests for SnowflakeFileTransferAgent.expandFileNames */
+public class FileUploaderSessionlessTest {
+  @Rule public TemporaryFolder folder = new TemporaryFolder();
+  private ObjectMapper mapper = new ObjectMapper();
+
+  private final String exampleJsonString =
```

Comment:
It'd be great to have strings across all three cloud providers as well, not sure how easy that is to pull off.